docs/common/input.md

```diff
@@ -460,6 +461,10 @@ parameters in this subsection are:
    not yet supported). This parameter is required and has no default value.
 * `tiny_h`, which is the water height below which a given point is assumed to
   be dry. Default value: `1e-7`
+* `h_anuga_reg_parameter`: is the water height used in velocity regularization using the
```

Comment:
It's actually the square of a height, right?

Reply:
Yeah, I'd rather the definition be $u = (hu)h / (h^2 + h_{\text{anuga}}^2)$ so that the regularization parameter has units of thickness. (This is a deviation from the paper, but I think they chose poor notation.)

Reply:
Oh, good point. 

How about I change the code from `denom = Square(data->h[n]) + h_anuga;` to
`denom = Square(data->h[n]) + Square(h_anuga);`?